TrackingTools/TrackAssociator/src/TrackDetectorAssociator.cc

```diff
@@ -804,9 +804,9 @@ void TrackDetectorAssociator::fillMuon( const edm::Event& iEvent,
                  }
               }
 	}
-	// GEM Chamber   
-	else if (parameters.useGEM){
-	  if(const GEMSuperChamber* chamber = dynamic_cast<const GEMSuperChamber*>(geomDet) ) {	 
+	// GEM Chamber or ME0 Chamber
+	else if (parameters.useGEM || parameters.useME0){
```

Comment:
This change seems to suggest that `useME0` can be false even though `geomDet` is an ME0 chamber. Why would that be so?